src/bindings/c/tests/ov_c_api_test.cpp

```diff
@@ -1,18 +1,17 @@
 // Copyright (C) 2018-2022 Intel Corporation
```

Comment:
Can we split this file on tests per component?

Reply:
ok, but I think we can put test case improvement into another PR to avoid this PR become too complex.